src/client/components/pages/MultiYearPlan.tsx

```diff
@@ -220,94 +224,127 @@ const MultiYearPlan: FunctionComponent = (): ReactElement => {
-                    isLabelVisible={false}
-                    hideError
-                    onChange={(event:React.ChangeEvent<HTMLInputElement>) => {
-                      setCourseTitleValue(event.currentTarget.value);
-                    }}
-                  />
-                </TableHeadingCell>
-                <>
-                  {instructorFilters(currentMultiYearPlanList, semesters)}
-                </>
-              </TableRow>
-            </TableHead>
-            <TableBody isScrollable>
-              {filteredMultiYearPlans()
-                .map((course, courseIndex): TableRow => (
-                  <TableRow isStriped={courseIndex % 2 === 1} key={course.id}>
-                    <TableCell
-                      verticalAlignment={VALIGN.TOP}
-                      backgroundColor={getCatPrefixColor(course.catalogPrefix)}
-                    >
-                      {course.catalogPrefix}
-                    </TableCell>
-                    <TableRowHeadingCell verticalAlignment={VALIGN.TOP} scope="row">
-                      {course.catalogNumber}
-                    </TableRowHeadingCell>
-                    <TableCell verticalAlignment={VALIGN.TOP}>
-                      {course.title}
-                    </TableCell>
-                    <>
-                      {courseInstance(course, semesters)}
-                    </>
-                  </TableRow>
-                ))}
-            </TableBody>
-          </Table>
-
+          <>
+            {WindowUtils.getHostname() === publicHostname
+              && (
+                <StyledInstructionalTextBox>
+                  <p>How to use this site:</p>
+                  <p>
+                    This site provides a snapshot of the current and 4-year
+                    course plan for courses offered by the Harvard School of
+                    Engineering and Applied Sciences. It also lists some
+                    specific courses in Earth and Planetary Sciences (EPS)
+                    that are typically taught by faculty with SEAS appointments.
+                  </p>
+                  <p>
+                    <b>
+                      Disclaimer: this course plan can change frequently and
+                      should be considered as a tentative, unofficial
+                      guideline only.
+                    </b>
+                  </p>
+                  <p>
+                    <a href="https://courses.my.harvard.edu/" target="_blank" rel="noreferrer">
+                      Courses.my.harvard.edu
+                    </a>
+                    {' '}
+                    is the official listing of courses. If a course is blank
+                    it means it does not have an assigned instructor yet
+                    and/or it is planned to be offered.
+                  </p>
+                  <p>Direct questions to: seas-course-planner@g.harvard.edu</p>
```

Comment:
Can we wrap this in an `<a href="mailto:seas-course-planner@g.harvard.edu">`?

Reply:
Good point! This was updated in [b171b46](https://github.com/seas-computing/course-planner/pull/529/commits/b171b46c7fd8c8daf918a0753d76dc151f5efe0d).